index.html

```diff
@@ -1,12 +1,147 @@
+        <p>@AU Assemble Hall, ADDIS ABABA ETHIOPIA</p>
+        </div>
+    </div>
+</section>
+</header>
+    <section class="mainpage">
+        <h1>Main Page</h1>
+         <hr class="redline">
+        <div class="boxes">
+        <div class="box">
+            <div class="photo"><img src="images/images (4).png" alt=""></div>
+           <h3>Lecture</h3>
+           <p>Listen from the speakers those who came from all over the world and who are profeciaonal in software development </p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (1).png" alt=""></div>
+            <h3>CC Exhibition </h3>
+            <p>Appreciate various creations applying software development licens of ariists organized from Art Center Nabi</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (2).png" alt=""></div>
+            <h3>Forum</h3>
+            <p>Have the same time to share to your thoughts, opinions with experts of software developers topic.</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (3).png" alt=""></div>
+            <h3>Workshop</h3> 
+            <p>Try Creating your own apps using open source license rather than just watching about software app.</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images.jpg" alt=""></div>
+            <h3>CC Ignite</h3>
+            <p>Get the  chance to network with CC affilliates around the world ,also after the summit of development</p>
+        </div>
+     </div>
+     <div class="redbox"> <h3> <a href="#">View full Program></a> </h3> </div>
+    </section>
+    <div class="feature">
+        <h1>Main Speakers</h1>
+        <hr class="redline"> 
+        <div class="bspeakers"></div>
+    </div>
+     <section class="sponcers">
+        <div class="platinum">
+          <h1>platinum Sponcers</h1>
+          <hr class="redline">
+          <div class="pltimage">
+          <img src="images/amazon.jpg" alt="amazon images">
+          <img src="images/google.jpg" alt="google images">
+          <img src="images/apple.jpg" alt="apple images">
+        </div>
+        </div>
+
+       <div class="gold"> 
+        <h1>Gold Sponcers</h1>
+        <hr class="redline">
+        <div class="goldimage">
+        <img src="images/channel logo.png" alt="channel images">
+        <img src="images/fedex.jpg" alt="fedex images">
+        <img src="images/dominos.jpg" alt="dominos images">
+        </div>
+       </div>
+    </section>
+    
+ 
+ <footer class="homefooter">
+
+    <div class="logo">
+      <img src="images/africalogo.webp" alt="logo images">
+    </div>
+     <div class="right">
+       <p>The african software development </p>
+       <p>@All right reserved</p>
+       <img src="images/social media icons.png" alt="social media images">
+    </div>
+
+</footer>
```

Comment:
- Just like the previous reviewer correctly pointed out, the background color for your footer image is different to the design template. Please fix this .

Your project:
<img width="950" alt="background capstone" src="https://user-images.githubusercontent.com/73607512/198673140-afe56e8a-d335-4e37-bea3-5fc2c7f914b8.PNG">

Design template:
<img width="622" alt="footer home behance" src="https://user-images.githubusercontent.com/73607512/198673411-35e06f8f-2862-4b58-8011-ba6839b0b587.PNG">